lib/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -175,7 +175,7 @@ a {
 
 ## Optional secondary options
 
-### `ignoreLonghands: ["string"]`
+### `ignoreLonghands`: `["array", "of", "properties"]|"property"`
```

Comment:
[suggest] Let's keep consistency in the heading:

```suggestion
### `ignoreLonghands: ["array", "of", "properties"]`
```

For example, see:
- https://github.com/stylelint/stylelint/blob/ca3c2f82d23d2629e987b246345cdeb80faf78a5/lib/rules/declaration-block-no-redundant-longhand-properties/README.md?plain=1#L232
- https://github.com/stylelint/stylelint/blob/ca3c2f82d23d2629e987b246345cdeb80faf78a5/lib/rules/declaration-block-no-duplicate-custom-properties/README.md?plain=1#L46